src/pages/CreateAccount/CreateAccount.jsx

```diff
@@ -1,15 +1,20 @@
 import React, { useState } from 'react';
-import { Form, Button } from 'antd';
+import PropTypes from 'prop-types';
+import { Form, Button, Steps } from 'antd';
 import { useForm, FormProvider } from 'react-hook-form';
 import * as yup from 'yup';
 import { yupResolver } from '@hookform/resolvers/yup';
-import GeneralInfo from '../components/create-account/GeneralInfo';
-import DuiAndCrimHis from '../components/create-account/DuiAndCrimHis';
-import RolesAndSkills from '../components/create-account/RolesAndSkills';
-import WeeklyInfo from '../components/create-account/WeeklyInfo';
-// import { AFCBackend } from '../util/utils';
+import GeneralInfo from '../../components/create-account/GeneralInfo/GeneralInfo';
+import DuiAndCrimHis from '../../components/create-account/DuiAndCrimHis/DuiAndCrimHis';
+import RolesAndSkills from '../../components/create-account/RolesAndSkills/RolesAndSkills';
+import WeeklyInfo from '../../components/create-account/WeeklyInfo/WeeklyInfo';
+import { AFCBackend } from '../../util/utils';
 
-const CreateAccount = () => {
+import styles from './CreateAccount.module.css';
+
+const { Step } = Steps;
+
+const CreateAccount = ({ setPageState, firstName, lastName, email }) => {
```

Comment:
Not sure why it won't leave me leave a comment for lines 26/27 where the regex expressions are, but can you move them to the common `utils` file and import them from there? Path to the utils file should be `../../util/utils` (not sure if I have the right number of `..`s in there but should be close)

Reply:
fixed!